src/test/components/MarkerChart.test.js

```diff
@@ -108,6 +108,7 @@ function setupWithProfile(profile) {
     ...renderResult,
     flushRafCalls,
     dispatch: store.dispatch,
+    getState: store.getState,
```

Comment:
I don't use it in the final patch, but I found it useful to have when debugging the tests.

Reply:
(No change needed) I just realized this recently, but you can also do `...store` here.